Buildings/Air/Systems/SingleZone/VAV/BaseClasses/ControllerChillerDXHeatingEconomizer.mo

```diff
@@ -313,6 +316,11 @@ heating coil and a cooling coil.
 </html>", revisions="<html>
 <ul>
 <li>
+November 20, 2020, by David Blum:<br/>
```

Comment:
This is a backwards-compatible bug fix and needs to be added to `Buildings/package.mo` in the release notes.

Reply:
I'm sorry, but I'm a little confused on how best to add to release notes given your comment here and also [this one](https://github.com/lbl-srg/modelica-buildings/pull/2266#discussion_r533476790).  This is a backwards-compatible bug fix, but the model is also moved and renamed.  I see in ``"new components have been added to existing libraries"`` models that have been moved and I see in ``"existing components have been improved in a non-backward compatible way"`` models that require conversion scripts.

Reply:
Added to release notes https://github.com/lbl-srg/modelica-buildings/pull/2266/commits/f7818c64c2bf782d4910ec5cf6c970a4ee679515.